### PR TITLE
[IMP][14.0] base_product_mass_addition: add support for UoM

### PR DESCRIPTION
**Pull request by kevinkhao**
required by https://github.com/OCA/purchase-workflow/pull/1046

### CODE REVIEW

File: base_product_mass_addition/models/product_product.py [hunk 1]
```diff
@@ -13,25 +13,53 @@ class ProductProduct(models.Model):
 
     qty_to_process = fields.Float(
         compute="_compute_process_qty",
-        inverse="_inverse_set_process_qty",
+        inverse="_inverse_quick_vals",
```
Comment (rousseldenis) [inline base_product_mass_addition/models/product_product.py]: @kevinkhao Is that change wanted as module is already merged ?

Comment (kevinkhao) [reply base_product_mass_addition/models/product_product.py]: thanks for feedback
reverted https://github.com/OCA/product-attribute/pull/864/commits/f9c49b8d193d248cfdf81e17f938cca793b09142

**Review by sebastienbeau — APPROVED** [review]
LGTM code review

**Review by bguillot — APPROVED** [review]
Tested

**Pull request merged**